tests/tests_suite_test.go

```diff
@@ -166,5 +166,10 @@ func resetToDefaultConfig() {
 		// we can just skip the restore step.
 		return
 	}
+	if ok, _ := CurrentSpecReport().MatchesLabelFilter("kvmanualreset"); ok {
```

Comment:
This reminds me that we probably don't want to blindly skip the serial case but assert the config have not changed.

Reply:
it did though. with my changes you'll never get equality here
(this is an aftereach as opposed to afterall doing the teardown)